service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -188,12 +244,7 @@ public ApiWorkspaceDescription createWorkspaceWithoutCloudContext(
             .andReturn()
             .getResponse()
             .getContentAsString();
-
-    // Return ApiWorkspaceDescription instead of ApiCreatedWorkspace, since former has
```

Comment:
Hmm I added this because a test needed userFacingId, which wasn't in ApiCreatedWorkspace. If this is no longer needed, fine to remove -- I can always add it back later if a test needs userFacingId.

Reply:
What I found was that the majority of tests did not need the description. Only the tests of userFacingId.
So I added the explicit `getWorkspace` into those tests. There are 4 and they are all very similar, so there may be a benefit in factoring part of the test out rather than putting the `getWorkspace` into the common `createWorkspace`.

Reply:
Ah, my thinking was that `getWorkspace` is fast so it was ok for every `createWorkspace`, but totally fine with this approach as well.